studygroups/decorators.py

```diff
@@ -56,7 +56,7 @@ def decorated(*args, **kwargs):
 
 def user_is_team_organizer(func):
     def decorated(*args, **kwargs):
-        team_id = kwargs.get('team_id')
+        team_id = kwargs.get('team_id') if kwargs.get('team_id', None) else kwargs.get('pk')
```

Comment:
You can set [`pk_url_kwarg`](https://docs.djangoproject.com/en/3.0/ref/class-based-views/mixins-single-object/#django.views.generic.detail.SingleObjectMixin.pk_url_kwarg) in `views.organizer.TeamUpdate` and then just use team_id here